ChurchToolsApi.py

```diff
@@ -633,4 +668,4 @@ def file_download_from_url(self, file_url: str, target_path: str):
                 return True
             else:
                 logging.warning("Something went wrong during file_download: {}".format(r.status_code))
-                return False
\ No newline at end of file
+                return False
```

Comment:
This is not related to the commit topic and should usually not be part of the commit - i suppose you can't exclude it anymore?